README.md

```diff
@@ -199,6 +200,20 @@ jfrogClient
   });
 ```
 
+#### Retrieving JAS configuration
+```javascript
+jfrogClient
+  .xray()
+  .jasconfig()
+  .getJasConfig()      
```

Comment:
```suggestion
  .jas()
  .getConfig()      
```
Maybe better?

Reply:
its not jas endpoint as shachar said so I prefer to leave it that way